repeater/src/main/scala/com.snowplowanalytics.snowplow.storage.bigquery.repeater/services/Storage.scala

```diff
@@ -29,12 +29,12 @@ object Storage {
 
   val TimestampFormat = DateTimeFormat.forPattern("YYYY-MM-dd-HHmmssSSS")
 
-  def getFileName(base: String, n: Int, tstamp: DateTime): String =
```

Comment:
Ouch, this one was an oversight. `getFileName` should be a pure function, accepting timestamp (created by `Clock`?) or should have return type wrapped in `IO`. Former one is obviously preferable.

Can live without it, but still makes me uncomfortable about such impurity.